### PR TITLE
1426: Add tax info for primary member

### PR DESCRIPTION
**Pull request by jessieay**
* We need some clarification before adding this information for
additional household members.
* [Part of #153062895]

https://www.pivotaltracker.com/story/show/153062895

Screenshot of this section of the 1426 from a test Medicaid app locally:

screen shot 2017-11-27 at 4 12 08 pm

### CODE REVIEW

File: app/models/medicaid_application_attributes.rb [hunk 1]
```diff
@@ -102,4 +111,12 @@ def residential_or_homeless
       "Homeless"
     end
   end
+
+  def dependents
```
Comment (hartsick) [inline app/models/medicaid_application_attributes.rb]: Would it make sense to have tests around this logic (filtering dependents & the first name display) at this object level?

File: app/models/medicaid_application_member_attributes.rb [hunk 2]
```diff
@@ -40,6 +40,10 @@ def to_h
         bool_to_checkbox(member.pay_student_loan_interest?),
       :"#{position}_member_deduction_alimony_yes" =>
         bool_to_checkbox(member.pay_child_support_alimony_arrears?),
+      joint_filer_key => "Yes",
```
Comment (hartsick) [inline app/models/medicaid_application_member_attributes.rb]: could we add object-level tests for logic about other joint filing member?

File: app/models/medicaid_application_attributes.rb (hunk 1 above)
Comment (jessieay) [reply app/models/medicaid_application_attributes.rb]: done at 42daaa993bb04faaa6c973436262c8bee6aa135d

File: app/models/medicaid_application_member_attributes.rb (hunk 2 above)
Comment (jessieay) [reply app/models/medicaid_application_member_attributes.rb]: done at 42daaa993bb04faaa6c973436262c8bee6aa135d

**Comment by jessieay** [comment]
Paired with Christa on this so mergin'

**Pull request merged**